contracts/test/OracleTest.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.7.6;
+pragma abicoder v2;
+
+import '@uniswap/v3-core/contracts/libraries/Oracle.sol';
+import '../libraries/OracleLibrary.sol';
+
+contract OracleTest {
```

Comment:
i'd rather not copy this in if we can avoid it by just testing directly against a pool with some helper functions for manipulating the price--one reason to do this is it would be more indicative of correctness of the oracle library

Reply:
Would using https://github.com/Uniswap/uniswap-v3-core/blob/main/contracts/test/MockTimeUniswapV3Pool.sol work? Unfortunately, I don't think it's exposed via NPM, and would still require copy and pasting the contract.

Reply:
oh, yeah i see, it depends on mock time. let me think about it some more, this is an ok approach for now.